basics/getting_started/update_plugins_format.md

```diff
@@ -0,0 +1,73 @@
+* Maintain an `updatePlugins.xml` file on that web server describing all the available plugins.
+* Add the URL for the custom repository to the JetBrains IDE [Repository Settings/Preferences](https://www.jetbrains.com/help/idea/managing-plugins.html#repos).
+
+## Describing Your Plugins in an updatePlugins File
+Every custom plugin repository must have at least one `updatePlugins.xml` file to describe the latest compatible version 
+for every hosted plugin. The description in `updatePlugins.xml` is used by JetBrains IDEs to locate plugins by attributes 
+such as id, IDE version, and plugin version. These attributes are displayed by JetBrains IDEs to help users select or upgrade plugins.
+The description also tells the JetBrains IDE where to download the plugin itself.
+
+A custom plugin repository's `updatePlugins.xml` file is constructed and maintained by the administrator of
+the repository. More than one `updatePlugins.xml` file may be required if consumers of the custom repository are using more
+than one version of a JetBrains IDE. For example `updatePlugins-182.xml`, `updatePlugins-183.xml` for IntelliJ IDEA 2018.2 and 2018.3 respectively.
+Each `updatePlugins-*.xml` file will have a unique URL that is added to the JetBrains IDE 
+[Repository Settings/Preferences](https://www.jetbrains.com/help/idea/managing-plugins.html#repos). 
+
+### The Format of an updatePlugins File
+The format of an `updatePlugins.xml` file is simply a list of sequential elements that describe each plugin:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 
+  The <plugins> element contains the description of the plugins available at this repository. Required. 
+-->
+<plugins>
+  <!-- 
+    Each <plugin> element describes one plugin in the repository. Required.
+    id - used by JetBrains IDEs to uniquely identify a plugin. Required. Must match <id> in plugin.xml
+    url - path to download the plugin JAR/ZIP file. Required. Must be HTTPS
+    version - version of this plugin. Required. Must match <version> in plugin.xml
+  -->
+  <plugin id="fully.qualified.id.of.this.plugin" url="https://www.mycompany.com/my_repository/mypluginname.jar" version="major.minor.update">
+    <!--
+      The <idea-version> element must match the same element in plugin.xml. Required.
+    -->
+    <idea-version since-build="181.3" until-build="184.*" />
+  </plugin>
+  <plugin id="id.of.different.plugin" url="https://www.otherserver.com/other_repository/differentplugin.jar" version="major.minor">
+    <idea-version since-build="181.3" until-build="184.*" />
+  </plugin>
+  <plugin>
+    <!-- And so on for other plugins... -->
+  </plugin>
+</plugins>
+```
+<br>
+**Note:** 
+* An `updatePlugins` file must contain at least one set of `<plugin></plugin>` elements.  
+* A plugin may be listed only once in an `updatePlugins` file.  
+* All plugins must be compatible with the same release(s) of the JetBrains IDE, or the plugin descriptions must be split into separate
```

Comment:
Is this actually a requirement? The IDE should be able to filter out incompatible plugins.